docs/pages/accounts/sso.mdx

```diff
@@ -51,9 +51,19 @@ SSO users are like regular users. However, there are a few known exceptions:
 
 ## SSO administration
 
-Both new organizations and existing organizations can enable SSO as a sign in option. Organizations with existing non-SSO members can enable SSO, and both non-SSO users and SSO users can continue to be added as members of SSO organizations.
+Both new organizations and existing organizations can enable SSO as a sign in option. Organizations with existing non-SSO members can enable SSO and then direct new members to the SSO sign-in page, while existing users continue to use their current Expo credentials. To support external contributors, SSO-enabled organizations also support inviting additional non-SSO users via email.
+
+### Transitioning existing users to SSO
+Compared to regular users, SSO users have an exclusive relationship with their organization. Thus, existing users cannot be directly migrated to SSO. However, a user that's already a member of your organization can create a second account by going to the [SSO login page](https://expo.dev/sso-login). Then, their old user can be removed from the organization.
+
+To transition from using a regular Expo account to an SSO account, follow these steps:
+1. Go to the [SSO login page](https://expo.dev/sso-login) and follow the prompts, entering your organization name, creating a new Expo username, and logging in to your identity provider.
+2. By default, your new SSO user will have the View Only role. If you need a different role, ask an Admin or Owner to update your role in [member settings](https://expo.dev/accounts/[account]/settings/members).
+3. At this time, the Admin or Owner can remove your old user from the organization. In [member settings](https://expo.dev/accounts/[account]/settings/members), the list of organization members indicates whether a user is an SSO or non-SSO user. The Admin or Owner can click the dropdown next to the old user and click **Remove member**.
+4. If you no longer need your old user account, log out of your new SSO account, then login to your old account and go to [user settings](https://expo.dev/settings). Scroll down and click **Delete Account**. **Note that this will delete any projects under  your old user account.** It will not affect any projects owned by the organization.
+
+> If you wish to reuse your old username on your new SSO user account, you can go to [user settings](https://expo.dev/settings) under your old user and rename it prior to creating your SSO account. You can also rename your SSO user account's Expo username later.
```

Comment:
```suggestion
> If you wish to reuse your old username on your new SSO user account, you can go to [user settings](https://expo.dev/settings) under your old user and rename it prior to creating your SSO account. Alternatively, you can rename your SSO user account's Expo username after deleting your old user.
```

Reply:
There was a comment from someone from Brex that they had to delete their existing user that used their work email before they could create their SSO user that uses their work email. I'm wondering if we need to explicitly say that you don't need to delete your existing account before creating an SSO user.

Reply:
@lzkb I was thinking about this, too, and wasn't sure where to slot it in without getting too wordy. I just tried adding it to the callout at the bottom of the "transition" section. Feedback most welcome!